src/test/java/api/UserApi.java

```diff
@@ -0,0 +1,30 @@
+package api;
+
+import com.github.javafaker.Faker;
+import io.restassured.response.Response;
+
+public class UserApi extends ApiBase {
+    RegistrationDTO dto;
+    Response response;
+    Faker faker = new Faker();
+
+    public RegistrationDTO randomDataForNewUser(){
+        dto = new RegistrationDTO();
+        dto.setFull_name(faker.name().username());
+        dto.setEmail(faker.internet().safeEmailAddress());
+        dto.setPassword("123456");
+        dto.setGenerate_magic_link("false");
+        return dto;
+    }
+    public Response registrationNewUserApi(Integer code, RegistrationDTO randomDataForRegisterUser){
+        String endpoint = "/v1/api/users";
+        String name = randomDataForNewUser().getFull_name();
+        response = doPostRequest(endpoint, code, randomDataForRegisterUser);
```

Comment:
replace randomDataForRegisterUser to randomDataForNewUser()

Reply:
done